pkg/apis/pipeline/v1beta1/customrun_types.go

```diff
@@ -194,7 +194,7 @@ func (r *CustomRun) GetStatusCondition() apis.ConditionAccessor {
 
 // GetGroupVersionKind implements kmeta.OwnerRefable.
 func (*CustomRun) GetGroupVersionKind() schema.GroupVersionKind {
-	return SchemeGroupVersion.WithKind(pipeline.RunControllerName)
+	return SchemeGroupVersion.WithKind(pipeline.CustomRunControllerName)
```

Comment:
We should call this out as a breaking change.

This is changing the GVK from `Run` -> `CustomRun`, which is going to break anything that is relying on this for GVK references (including owner refs for deletion).
That said, this is also a bug fix for v1beta1.CustomRun since the [correct kind is `CustomRun`](https://github.com/tektoncd/pipeline/blob/c46925a458b9444d4763333e0a97cfd192b7721c/config/300-customrun.yaml#L60). 😬 

We should probably include documentation in the release notes warning people about the bug and how they should go about identifying incorrect references for v1beta1.CustomRun. OwnerRefs are probably going to be the most common - we may want to consider making a small tool to identify bad references.

Reply:
Ohh okay.. sounds good! I've reverted the changes from `CustomRunControllerName` to `RunControllerName`. Thanks so much!

Reply:
@wlynch do you mean we should fix the bug, or fix it and add a note in the release notes? 
`CustomRun` is a `v1beta1` API but I wouldn't feel ok waiting for many months for fixing this.

Reply:
@tektoncd/core-collaborators FYI

Reply:
> @wlynch do you mean we should fix the bug, or fix it and add a note in the release notes?

Fix it and call it out in the release notes as a breaking - it's a bug fix, not a feature deprecation so I don't think we need to wait. If the alpha type needs this as Run, then we should fork the type.

Reply:
That said, I don't see a reason to hold up this PR - we can follow up in another change.